src/main/java/frc/robot/RobotContainer.java

```diff
@@ -69,28 +80,44 @@ private void configureButtonBindings() {
 
     // turns on shooter when Left Bumper is pressed
     new JoystickButton(m_operatorController, Button.kBumperLeft.value)
-        .whenPressed(new ShooterOnCommand(m_shooterSubsystem));
+       .whenPressed(new ShooterOnCommand(m_shooterSubsystem));
```

Comment:
formatting this is outside the scope of the PR
```suggestion
        .whenPressed(new ShooterOnCommand(m_shooterSubsystem));
```